egs/wsj/s5/steps/durmod/durmodel_train.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
```

Comment:
I would recommend renaming this file to just train.sh as it is already located in the duration model directory.